v-next/hardhat/package.json

```diff
@@ -68,6 +68,7 @@
     "@types/debug": "^4.1.4",
     "@types/node": "^20.14.9",
     "@types/semver": "^7.5.8",
+    "@types/toposort": "^2.0.7",
```

Comment:
We have an implementation of topological sort, which we should reuse

Reply:
I removed the dependency and just used the dfs approach for walking and sorting the files. The order is deterministic but it's different than in v2. As discussed in the call, we are ok with that change.

Reply:
@antico5 Let's remove the `@types/toposort` dependency, too.